pkg/reconciler/broker/controller.go

```diff
@@ -53,6 +56,10 @@ import (
 	"knative.dev/pkg/resolver"
 )
 
+const (
+	component = "rabbitmq_dispatcher"
```

Comment:
sure would be nice to use find a way to not duplicate this to various places, but we can decide to that later on.